### ISSUE TITLE
Resiliency example constant retry using incorrect maxInterval option?

### ISSUE THREAD

**Issue by LordMsz** [dapr/quickstarts#772]
## Ask your question here
I'd like to ask about `maxInterval` option used for `constant` policy in retries examples, such as here:
https://github.com/dapr/quickstarts/blob/master/service_invocation/components/resiliency.yaml#L13

I was very confused about how it worked for me locally. Things retried in very weird intervals and not as I expected.

After a lot of fiddling and reading [docs](https://docs.dapr.io/operations/resiliency/policies/#retries) many times I finally realized that `constant` policy completely ignores `maxInterval` option and that `duration` should be used instead? So actual duration used in the samples is 5 seconds (default)?
Or am I doing / understanding something completely wrong?

If my assumption is right, then the samples are very confusing and could mislead many like me.

I'd be willing to make a PR if you're interested and it would help.

Thanks for all the great work! =)

**Some details of my setup if relevant:**
 - running docker desktop selfhosted on Windows 11
 - I have my own C# .NET 6 application; but it is basically same as CSharp resiliency example https://github.com/dapr/quickstarts/blob/master/resiliency/service-to-service-resiliency.md

**Comment by greenie-msft** [comment]
@LordMsz sorry for the delayed response. Good catch with this. The project would be more than happy to take an incoming from you to fix this issue!

Could you please fix both the service invocation and state quickstarts? 

Thank you!

**Comment by LordMsz** [comment]
Hello, no problem :)

Sure, I tried, and hope I didn't miss anything, PR is in #779 
Tried running the quickstarts locally and, as expected, it worked.

Interestingly, running the command from documentation didn't work for me, but that's irrelevant to this issue.
> dapr run --app-port 7001 --app-id order-processor --config ../config.yaml --components-path --app-protocol http --dapr-http-port 3501 -- dotnet run
This was throwing that "--app-protocol: system can't find path specified".
I just added "..\" after --components-path and it worked. Not sure it's an issue or my setup.
